crates/storage-proto/proto/dev/restate/storage/v1/domain.proto

```diff
@@ -375,11 +375,14 @@ message OutboxMessage {
 // ---------------------------------------------------------------------
 
 message Timer {
-    bytes service_name = 1;
-    bytes service_key = 2;
+
+    message CompleteSleepEntry {
+        bytes service_name = 1;
+        bytes service_key = 2;
+    }
 
     oneof value {
-        google.protobuf.Empty complete_sleep_entry = 100;
+        CompleteSleepEntry complete_sleep_entry = 100;
```

Comment:
Did we already bump the storage format version for the upcoming release? If not, then this needs to be done since we are changing the storage format.

Reply:
Done